### ISSUE TITLE
Set the values in contract function after contract creation

### ISSUE THREAD

**Issue by Pris17** [ConsenSys/eth-lightwallet#148]
I am using eth-lightwallet to generate accounts and then sign and create contract. I am able to create a contract and send raw transaction to blockchain node. Now i want to call one of the contract function which accepts 4-5 arguements and sets those arguements in variables. But I am not able to do this. Getting error as "Cannot read property 'inputs' of undefined".. Any help appreciated

**Comment by Priya-Nets** [comment]
@christianlundkvist  Please provide a solution for this.

**Comment by sc0Vu** [comment]
Hi @Pris17 

Did you mean this?

     var contract = web3.eth.contract(abi).at(.contractAddress);
     var data = contract.yourFunctionName([args]);

**Comment by Pris17** [comment]
@sc0vu No. Not this way. I have use lightwallet methods to set the values in contract which doesn't seems to work.
